### ISSUE TITLE
Translucent extrusions don't work correctly

### ISSUE THREAD

**Issue by TimSylvester** [maplibre/maplibre-native#1589]
In legacy GL, translucent extrusions mask others behind them.  In drawable GL and Metal builds, the back ones mask the front ones.  Seems related to the stencil test.

<img src="https://user-images.githubusercontent.com/71895881/264470807-7215e1ee-9bea-4fd2-b00f-ab02f607f66a.gif" width="200"> <img src="https://user-images.githubusercontent.com/71895881/264470408-a95a76dd-8634-4b83-b144-b10ed3e9d89e.gif" width="200"> <img src="https://user-images.githubusercontent.com/71895881/264470449-1367515b-cd58-4fe0-8d76-623218f72d2c.gif" width="200">

**Comment by alanchenboy** [comment]
I think, this is because for transparent objects need accurate render order from behind to front. when draw extrusion stencil is disabled in mapbox.

**Comment by TimSylvester** [comment]
Yes, the only difference here seems to be draw ordering.  If we enforced the same order in drawables I think we would get identically incorrect results.

When stenciling isn't used, this is handled with depth testing.

**Comment by sjg-wdw** [comment]
We've resolved this I think?

**Comment by TimSylvester** [comment]
No, but we didn't cause it, either.  It would require drawing tiles/features in a different order depending on the view.
